benchmark/src/txn_generator.rs

```diff
@@ -28,6 +29,31 @@ const TXN_EXPIRATION: i64 = 100;
 /// due to short of balance error in generated transfer TXNs.
 const FREE_LUNCH: u64 = 1_000_000;
 
+/// To unify both write and read requests.
+// Current Benchmarker only support submit and check WriteRequest.
+pub enum LibraRequest {
+    WriteRequest(SubmitTransactionRequest),
+    ReadRequest(UpdateToLatestLedgerRequest),
+}
+
+/// This interface specifies the requirements to generate customized TXN load
+/// and read requests in order to run within benchmark library.
+/// Required methods are expected to be called in specified order.
+pub trait LibraRequestGenerator {
```

Comment:
What about renaming `LibraRequestGenerator` to `LoadGenerator`? I think `Libra` prefix is redundant given that the whole repo is for Libra, and `RequestGenerator` is a little confusing on whether it's generating a single request or many requests.

Reply:
Yeah, it takes me some time to figure out a name. `LoadGenerator` sounds good to me.